DirectProgramming/DPC++FPGA/Tutorials/Features/latency_control/src/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE latency_control.cpp)
+set(TARGET_NAME latency_control)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(FPGA_TARGET ${TARGET_NAME}.fpga)
+
+# FPGA board selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "intel_a10gx_pac:pac_a10")
+    message(STATUS "FPGA_BOARD was not specified.\
+                    \nConfiguring the design to run on the default FPGA board ${FPGA_BOARD} (Intel(R) PAC with Intel Arria(R) 10 GX FPGA). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA board ${FPGA_BOARD}")
+endif()
+
+# This is a Windows-specific flag that enables exception handling in host code
+if(WIN32)
+    set(WIN_FLAG "/EHsc")
+endif()
+
+# A DPC++ ahead-of-time (AoT) compile processes the device code in two stages.
+# 1. The "compile" stage compiles the device code to an intermediate representation (SPIR-V).
+# 2. The "link" stage invokes the compiler's FPGA backend before linking.
+#    For this reason, FPGA backend flags must be passed as link flags in CMake.
+set(EMULATOR_COMPILE_FLAGS "-Wall ${WIN_FLAG} -fintelfpga -DFPGA_EMULATOR")
```

Comment:
Robert Ho uses an extra include "-I${SDK_ROOT_PATH}/include/sycl/ext/intel/prototype" for his host pipes tutorial. Why don't you need it? Are this experimental pipe implementation in a different place that Robert's experimental pipes implementation?

Reply:
Rob's host_pipe.hpp was submitted to our PSG's Perforce depot, and will be delivered within our FPGA backend tarball.

My latency control header was submitted to the public [invel/llvm ](https://github.com/intel/llvm) repo, and will be delivered within DSE's DPCPP frontend tarball along with all other extensions included in `<sycl/ext/intel/fpga_extensions.hpp>`.